unikraft/app/application.go

```diff
@@ -784,3 +787,42 @@ func (app application) Save() error {
 
 	return nil
 }
+
+func (app application) RemoveLibrary(ctx context.Context, libraryName string) error {
+	for libKey, lib := range app.libraries {
+		if lib.Name() == libraryName {
+			var kraftfileValues application
+			yamlFile, err := os.ReadFile(app.kraftfile.path)
+			if err != nil {
+				return err
+			}
+			err = yaml.Unmarshal(yamlFile, &kraftfileValues)
+			if err != nil {
+				return err
+			}
+			delete(app.libraries, libKey)
+			delete(kraftfileValues.libraries, libraryName)
+
+			yaml, err := yaml.Marshal(&kraftfileValues)
+			if err != nil {
+				return err
+			}
+
+			// Write the YAML data to the file
+			err = os.WriteFile(app.kraftfile.path, []byte(yaml), 0o644)
```

Comment:
I think you should preserve the permissions of the old file that you read above. You can get them with `os.Stat`

Reply:
Done 👍🏽